lib/streamlit/bootstrap.py

```diff
@@ -87,6 +86,39 @@ def _fix_matplotlib_crash():
             pass
 
 
+def _fix_tornado_crash():
+    """set default asyncio policy to be compatible with tornado
+    Tornado 6 (at least) is not compatible with the default
+    asyncio implementation on Windows
+    Pick the older SelectorEventLoopPolicy on Windows
+    if the known-incompatible default policy is in use.
+    do this as early as possible to make it a low priority and overrideable
+    ref: https://github.com/tornadoweb/tornado/issues/2608
+    FIXME: if/when tornado supports the defaults in asyncio,
+           remove and bump tornado requirement for py38
+    """
+    if sys.platform.startswith("win") and sys.version_info >= (3, 8):
```

Comment:
Please use the new `env_util.IS_WINDOWS` from `streamlit.env_util` :smiley:

Reply:
i didn't find any env_util in the code